doc/command_ref.rst

```diff
@@ -979,15 +979,30 @@ Module subcommands take :ref:`\<module-spec>\ <specifying_modules-label>`... arg
     In case no profile was provided, all default profiles get installed.
     Module streams get enabled accordingly.
 
-    This command cannot be used for switching module streams. It is recommended to remove all
-    installed content from the module and reset the module using the
-    :ref:`reset <module_reset_command-label>` command. After you reset the module, you can install
-    the other stream.
+    This command cannot be used for switching module streams. It is recommended to use the
+    :ref:`dnf module switch <module_switch_command-label>` command.
 
 ``dnf [options] module update <module-spec>...``
     Update packages associated with an active module stream, optionally restricted to a profile.
     If the `profile_name` is provided, only the packages referenced by that profile will be updated.
 
+.. _module_switch_command-label:
+
+``dnf [options] module switch <module-spec>...``
+    Switch or enable module stream, change a version of installed packages to a version provided by
+    the new stream, and remove packages from the old stream that are no longer available. It also
+    updates installed profiles if they are available for the new stream. In case that <module-spec>
+    contains profile, it installs that profile and does not update already installed profiles.
```

Comment:
> In **case <module-spec>** contains **a** profile, it installs that profile and does not update **any** already installed profiles.

Reply:
I modified it with an additional change.